.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__
 # Other generated files
 htmlcov
 .coverage
+py/desiutil/.cache
```

Comment:
This file was generated by running `pytest --doctest-modules py/desiutil/plots.py`, which isn't normally invoked by travis or `python setup.py test` (but perhaps should be?)  In any case, I just checked that `pytest` runs fine when `py/desiutil/` is read only.